LICENSE-APACHE

```diff
@@ -178,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
```

Comment:
Please do not change the files within the root.  They are maintained in https://github.com/epage/_rust